### ISSUE TITLE
Filter Params resolves Feature Request 6141

### ISSUE THREAD

**Issue by emyoulation** [kkujansuu/gramps#43]
006141: Define Custom Filter: 'do not match filter' also per rule
https://gramps-project.org/bugs/view.php?id=6141

**Comment by kkujansuu** [comment]
I don't think so. The 'invert' checkbox still applies only at the filter level.
